source/1-SDLC-organization/lib/cdk-bootstrap-template.yml

```diff
@@ -424,7 +418,7 @@ Resources:
               - s3:GetEncryptionConfiguration
               - s3:PutEncryptionConfiguration
             Resource:
-              - "*"
+              - !Sub "arn:aws:s3:::*"
```

Comment:
S3 Bucket cannot have account in the ARN. There is also no condition key to limit this to buckets in the current account.

https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazons3.html